go/multiorch/recovery/actions/bootstrap_shard_test.go

```diff
@@ -350,13 +350,14 @@ func TestBootstrapShardAction_ConfiguresSyncReplication(t *testing.T) {
 	assert.Len(t, mockClient.syncReplicationRequest.StandbyIds, 2,
 		"Should configure both standbys in the sync replication list")
 
-	// Verify the standby IDs are correct (order doesn't matter)
+	// Verify the standby IDs are valid pooler names (primary selection is non-deterministic)
```

Comment:
@cuongdo  this should fix this flake: https://github.com/multigres/multigres/actions/runs/20077317166/job/57595035056?pr=333#step:6:632